packages/ubuntu_wsl_setup/lib/app.dart

```diff
@@ -13,36 +14,46 @@ class UbuntuWslSetupApp extends StatelessWidget {
     super.key,
     this.variant,
     this.initialRoute,
+    this.showSplashScreen = false,
   });
 
   final Variant? variant;
   final String? initialRoute;
+  final bool showSplashScreen;
 
   @override
   Widget build(BuildContext context) {
-    return InheritedLocale(
-      child: Builder(builder: (context) {
-        return MaterialApp(
-          locale: InheritedLocale.of(context),
-          onGenerateTitle: (context) {
-            final lang = AppLocalizations.of(context);
-            setWindowTitle(lang.windowTitle);
-            return lang.appTitle;
-          },
-          theme: yaruLight,
-          darkTheme: yaruDark,
-          debugShowCheckedModeBanner: false,
-          localizationsDelegates: localizationsDelegates,
-          supportedLocales: supportedLocales,
-          home: buildWizard(context),
-        );
-      }),
+    return ColoredBox(
+      color: Colors.white,
```

Comment:
should it respect the theme? perhaps this could be moved inside SplashScreen and use `Theme.of(context).backgroundColor`?

Reply:
oh wait, but we don't have a dark variant of the gif :rofl:

Reply:
i think i'd have this in SplashScreen regardless because it's specifically for the splash screen. does it cause any issues that way?

Reply:
Being on the splash screen doesn't change anything 😅

Reply:
Maybe I should make the splash the top level one. I avoided that because it's a conditional thing.

Reply:
Ha! Figured it out! I only need the AnimatedSwitcher to be child of the ColoredBox, that's enough. Let me fix it.

Reply:
This is the result. 

https://user-images.githubusercontent.com/11138291/186925883-93ae0bb5-c354-4ec7-9d49-1aa287887739.mp4

Reply:
Smooth!

Reply:
Thank you for pointing that out.